admin_manual/installation/installation_wizard.rst

```diff
@@ -170,8 +178,13 @@ and files:
 * The :file:`apps/` directory should be owned by ``[HTTP user]:[HTTP group]``
 * The :file:`config/` directory should be owned by ``[HTTP user]:[HTTP group]``
 * The :file:`themes/` directory should be owned by ``[HTTP user]:[HTTP group]``
+* The :file:`assets/` directory should be owned by ``[HTTP user]:[HTTP group]``
 * The :file:`data/` directory should be owned by ``[HTTP user]:[HTTP group]``
 * The :file:`[ocpath]/.htaccess` file should be owned by ``root:[HTTP group]``
 * The :file:`data/.htaccess` file should be owned by ``root:[HTTP group]``
 * Both :file:`.htaccess` files are read-write file owner, read-only group and 
   world
+
+If you want to update your installation via the GUI or the occ-command,
+check :ref:`setting_strong_permissions` in the Update section.
```

Comment:
We do have the following, please correct me if I am wrong:
Setting Strong Directory Permissions
------------------------------------
in: ``admin_manual/installation/installation_wizard.rst``

and

Setting Strong Permissions
--------------------------
in: ``admin_manual/maintenance/update.rst``

I want to notify the reader of the ``installation_wizard.rst`` that he should consider reading the update document, the correspronding section for permissions, when updating/upgrading his instance.

BTW: I find it a bit confusing having twice more or less same title. But I do not know where/if the target refrence link is used elsewhere, so I refused to change its name to a better reading one...

I would suggest changing in the updater document the link_name 
``.. _setting_strong_permissions:`` to ``.. _setting_permissions_for_updating:``
and the heading text 
``Setting Strong Permissions`` to ``Setting Permissions for Updating``
 
and then reference accordingly.

Please advise